README.md

```diff
@@ -4,6 +4,36 @@
 
 Tools for implementing SCSI on top of NVMe
 
+This repo contains the following:
+1) SCSI mock device
+2) NVMe device driver
+3) Translation Library
+4) Engine to orchestrate execution flow
+
+## End-to-end translation ##
+
+### Setup ###
+1) Call `$ make` in the root directory. Ensure this results in a scsi2nvme.ko
+2) Call `$ insmod scsi2nvme.ko` to insert the kernel module.
+
+    Upon inserting, the SCSI mock device will be automatically attached. Once the device is attached, the SCSI subsystem will issue a set of commands. 
+
+    `$ lsblk ` should show the device at /dev/sdb
```

Comment:
The exact directory at which the device is shown varies based on how many SCSI disks are attached to the machine